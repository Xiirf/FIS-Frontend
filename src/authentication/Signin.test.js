import React from 'react';
import Signin from './Signin';
import { render, unmountComponentAtNode } from 'react-dom';
import { create } from "react-test-renderer";

let container = null;
beforeEach(() => {
  // met en place un élément DOM comme cible de rendu
  container = document.createElement("div");
  document.body.appendChild(container);
});

afterEach(() => {
  // nettoie en sortie de test
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

it('renders without crashing', () => {
  render(<Signin />, container);
});

it("matches the snapshot", () => {
    const signin = create(<Signin />);
    expect(signin.toJSON()).toMatchSnapshot();
});