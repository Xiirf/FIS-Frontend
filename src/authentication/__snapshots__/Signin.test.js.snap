// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div>
  <h2>
    Ingresar
  </h2>
  <form
    action="#"
    onReset={[Function]}
    onSubmit={[Function]}
  >
    <div
      className="form-group"
    >
      <label
        htmlFor="username"
      >
        Nombre de usuario
      </label>
      <input
        className="form-control"
        name="username"
        onBlur={[Function]}
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="form-group"
    >
      <label
        htmlFor="password"
      >
        Contrase√±a
      </label>
      <input
        className="form-control"
        name="password"
        onBlur={[Function]}
        onChange={[Function]}
        type="password"
        value=""
      />
    </div>
    <div
      className="form-group"
    >
      <button
        className="btn btn-primary"
        disabled={false}
        type="submit"
      >
        Ingresar
      </button>
    </div>
  </form>
</div>
`;
